package-lock.json

```diff
@@ -74,7 +74,7 @@
                 "socket.io": "~4.6.1",
                 "socket.io-client": "~4.6.1",
                 "socks-proxy-agent": "6.1.1",
-                "sqlite3": "^5.1.7",
+                "sqlite3": "~5.1.7",
```

Comment:
Why?

And why not changed package.json?

Reply:
https://github.com/louislam/uptime-kuma/blob/4794f9eb0bc47ebfe3eed3edaa7f880430052543/package.json#L142